libs/features/components/cards/src/lib/pages/elewa-group-card/elewa-group-card.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ElewaGroupCardComponent } from './elewa-group-card.component';
+
+describe('ElewaGroupCardComponent', () => {
+  let component: ElewaGroupCardComponent;
+  let fixture: ComponentFixture<ElewaGroupCardComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ElewaGroupCardComponent],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(ElewaGroupCardComponent);
```

Comment:
same

styling for this component has been deleted as well

Reply:
I don't understand what you mean

Reply:
Since this was my first issue and was merged before i could send a pull request, i pulled the changes from main branch in github